location-service/src/main/java/com/movie/recommender/location/dto/LocationsDTO.java

```diff
@@ -0,0 +1,54 @@
+package com.movie.recommender.location.dto;
+
+import com.movie.recommender.location.model.entity.Locations;
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.DecimalMin;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import java.time.LocalDateTime;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class LocationsDTO {
```

Comment:
separate DTO for creation and returning data

`LocationCreateDTO` used when sending data to the API for creating a new locatiotion
this DTO does not include fields like `id` or `createdAt`, which are not provided by the client during creation

`LocationDTO` used for returning data to the client
this DTO can include `id` and `createdAt` since these fields are generated after the entity is persisted